network/core-node/admin-guide/monitoring.mdx

```diff
@@ -125,34 +142,143 @@ This list is the result of both inbound connections from other peers and outboun
 }
 ```
 
-</CodeExample>
+### Overlay Topology Survey
+
```

Comment:
In Protocol 21, there are some changes coming to survey script. I will let you know in a week or two so we can update this section.